app/services/geek-jokes.js

```diff
@@ -0,0 +1,21 @@
+const axios = require('axios');
+
+const getJoke = () => {
+  axios
```

Comment:
You are not returning the promise. You will have a problem with the asyncronism.